winter/controller.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, func, url_path: str, http_method: str):
         self.func = func
         self.url_path = url_path
         self.http_method = http_method
-        self._arguments = self._build_arguments(func)
+        self._arguments, self.return_value_class = self._build_arguments(func)
```

Comment:
It's not straightforward to return `return_value_class` from `build_arguments`. May be pass `arguments_type_hints` to `_build_arguments`.

Reply:
done